disttask/framework/dispatcher/dispatcher.go

```diff
@@ -50,6 +50,12 @@ type Dispatch interface {
 	Stop()
 }
 
+// Handle provides the interface for operations needed by task flow handles.
+type Handle interface {
+	// GetTaskAllInstances gets handles the task's all available instances.
+	GetTaskAllInstances(ctx context.Context, gTaskID int64) ([]string, error)
```

Comment:
> gTaskID

Handle is per task/dispatcher node/cluster?

Reply:
per dispatcher